internal/pkg/routes/route.go

```diff
@@ -4,23 +4,46 @@
 package routes
 
 import (
+	"fmt"
+
 	mqtt "github.com/eclipse/paho.mqtt.golang"
 	"github.com/mainflux/mainflux/logger"
 	"github.com/nats-io/nats.go"
 )
 
-type R struct {
+// Route - message route, tells which nats topic messages goes to which mqtt topic.
+// Later we can add direction and other combination like ( nats-nats).
+// Route is used in mfx and plain. Route is like base implementation and mfx and plain
+// are extended implemntation.
+type Route struct {
```

Comment:
Do you need this struct here at all?

Do you need interface implementation here at all, or this will be the role of whoever imports this interface?